tests/utils/shopper.js

```diff
@@ -2,15 +2,26 @@
  * External dependencies
  */
 import { shopper as wcShopper } from '@woocommerce/e2e-utils';
-
 /**
  * Internal dependencies
  */
+import { sleep } from '../e2e/utils';
 import { getBlockPagePermalink } from './get-block-page-permalink';
 
 export const shopper = {
 	...wcShopper,
 
+	goToCartBlock: async () => {
+		const checkoutBlockPermalink = await getBlockPagePermalink(
+			`Cart Block`
+		);
+
+		await page.goto( checkoutBlockPermalink, {
+			waitUntil: 'networkidle0',
+		} );
+		await page.waitForSelector( 'h1', { text: 'Cart' } );
```

Comment:
I know this is something present in other tests, but the API for waitForSelector does not allow to pass text here. https://github.com/puppeteer/puppeteer/blob/main/docs/api.md#pagewaitforselectorselector-options

Better to use `await expect(page).toMatchElement('h1', { text: 'Cart' })` from expect-pupetter.